apps/element-storybook/.storybook/main.ts

```diff
@@ -1,17 +1,19 @@
+import { dirname, join } from "path";
 import remarkGfm from 'remark-gfm';
 import { StorybookConfig } from '@storybook/react-vite';
 
 const config: StorybookConfig = {
   stories: [
     '../src/*.mdx',
-    '../src/**/*.stories.@(js|jsx|ts|tsx|mdx)',
+    '../src/**/*.@(stories.@(js|jsx|ts|tsx))',
```

Comment:
we still need *.stories.mdx for the bootstrap migration files, although it looks like the component migration template had it's filename changed so it would also be caught in this. Was it causing an error or can we change the name back to `bs4migration.mdx`?

Reply:
It was causing an error. Something about Storybook not supporting mdx2. The error recommended adding .stories

Reply:
aha, I got it working without .stories